models/_sector/dex/bot_trades/solana/platforms/consortium_key_solana_bot_trades.sql

```diff
@@ -0,0 +1,110 @@
+        unique_key=[
+            "blockchain",
+            "tx_id",
+            "tx_index",
+            "outer_instruction_index",
+            "inner_instruction_index",
+        ],
+    )
+}}
+
+{% set project_start_date = "2024-03-01" %}
+{% set fee_receiver_1 = "6qgwjhV2RQxcPffRdtQBTTEezRykQKXqhcDyv1z3r9tq" %}
+{% set wsol_token = "So11111111111111111111111111111111111111112" %}
+
+with
+    users as (
+        select distinct transactions.signer as user
+        from {{ source("solana", "account_activity") }} as activity
+        join {{ source("solana", "transactions") }} as transactions on tx_id = id
+        where
+            activity.block_time >= timestamp '{{project_start_date}}'  -- FirstTradeTimestamp
+            and transactions.block_time >= timestamp '{{project_start_date}}'  -- FirstTradeTimestamp
+            and tx_success
+            and address = '{{fee_receiver_1}}'  -- SolanaFeeWallet
+            and balance_change > 0  -- SOL fee payments
+    ),
+    bottrades as (
+        select
+            trades.block_time,
+            cast(date_trunc('day', trades.block_time) as date) as block_date,
+            cast(date_trunc('month', trades.block_time) as date) as block_month,
+            'solana' as blockchain,
+            amount_usd,
+            if(token_sold_mint_address = '{{wsol_token}}', 'Buy', 'Sell') as type,
+            token_bought_amount,
+            token_bought_symbol,
+            token_bought_mint_address as token_bought_address,
+            token_sold_amount,
+            token_sold_symbol,
+            token_sold_mint_address as token_sold_address,
+            project,
+            version,
+            token_pair,
+            project_program_id as project_contract_address,
+            trader_id as user,
+            trades.tx_id,
+            tx_index,
+            outer_instruction_index,
+            inner_instruction_index
+        from {{ ref("dex_solana_trades") }} as trades
+        join users on trader_id = user
+        where
+            {% if is_incremental() %} {{ incremental_predicate("trades.block_time") }}
+            {% else %} trades.block_time >= timestamp '{{project_start_date}}'
+            {% endif %}
+    ),
+    highestinnerinstructionindexforeachtrade as (
+        select
+            tx_id,
+            outer_instruction_index,
+            max(inner_instruction_index) as highestinnerinstructionindex
+        from bottrades
+        group by tx_id, outer_instruction_index
+    )
+select
+    block_time,
+    block_date,
+    block_month,
+    blockchain,
+    amount_usd,
+    type,
+    token_bought_amount,
+    token_bought_symbol,
+    token_bought_address,
+    token_sold_amount,
+    token_sold_symbol,
+    token_sold_address,
+    project,
+    version,
+    token_pair,
+    project_contract_address,
+    user,
+    bottrades.tx_id,
+    tx_index,
+    bottrades.outer_instruction_index,
+    coalesce(inner_instruction_index, 0) as inner_instruction_index,
```

Comment:
Maybe we can try this way, and also add them in the schema please:
```suggestion
    tx_index,
    bottrades.outer_instruction_index,
    CAST(NULL AS DOUBLE) as fee_usd,
    CAST(NULL AS DOUBLE) as fee_token_amount,
    coalesce(inner_instruction_index, 0) as inner_instruction_index,
```